src/components/profileEdit/AddressInfo.tsx

```
@@ -137,43 +179,48 @@ export default function AddressInfo() {
-                            type="text"
-                            value={newAddress.detailAddress}
-                            onChange={(e) => setNewAddress({
-                                ...newAddress,
-                                detailAddress: e.target.value,
-                            })}
-                            className="w-full px-3 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-main-color"
-                        />
+                        <div>
+                            <TextInput
+                                value={newAddress.detailAddress}
+                                onChange={handleChangeDetailAddress}
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)